chart/templates/deployment.yaml

```diff
@@ -79,6 +79,10 @@ spec:
                   name: {{ include "obot.config.secretName" . }}
                   key: githubAuthToken
             {{- end }}
+            - name: "OBOT_SERVER_ENABLE_AUTHENTICATION"
+              value: "{{ .Values.config.enableAuthentication }}"
```

Comment:
just as a note, most of these keys are just the camelcase version of the env var, so this one doesn't match the rest right now

Reply:
I updated the bootstrap user one to put `obotServer` in front to make it consistent